homeassistant/components/hausbus/device.py

```diff
@@ -0,0 +1,158 @@
+"""Representation of a Haus-Bus device."""
```

Comment:
this belong to the library.

Reply:
I'm not saying the classes should just be dropped in the lib as-is. It just seems to me, like there's lot of things in those 3 files (like the binary mapping in this file) that are not really HA specific. Bases for all entities are usually placed in a file called `entity.py`

Reply:
I guess you are right about the mapping. I will ask the library owner to move it there.

The device class is not the base for all entities. These are devices that can have multiple entities. It is not a one to one relation for devices to entities.

Reply:
Moved the mapping to the pyhausbus library